### PR TITLE
v0.0.1: Development <- Dev-IMJ

### PR DESCRIPTION
**Pull request by notHooman996**

### CODE REVIEW

File: frontend/example-game.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+const scene = new THREE.Scene();
+const camera = new THREE.PerspectiveCamera( 75, window.innerWidth / window.innerHeight, 0.1, 1000 );
+
+const renderer = new THREE.WebGLRenderer();
+renderer.setSize( window.innerWidth, window.innerHeight );
```
Comment (lmbek) [inline frontend/example-game.js:5]: Great! Now we can wrap it in a class in the future.

File: go.mod [hunk 2]
```diff
@@ -0,0 +1,5 @@
+module PingPong
+
+go 1.19
+
```
Comment (lmbek) [inline go.mod:4]: Great

File: main.go [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+var frontendPath = "./frontend" // this should be set to where frontend files is (frontend folder: html, css, javascript...)
+
+// remember to change the ports to something unique
+var chromeLauncher = launcher.ChromeLauncher{
+	Location:                "C:\\Program Files\\Google\\Chrome\\Application\\chrome.exe",
+	LocationCMD:             "C:\\\"Program Files\"\\Google\\Chrome\\Application\\chrome.exe",
+	FrontendInstallLocation: os.Getenv("localappdata") + "\\Google\\Chrome\\InstalledApps\\" + organisationName + "\\" + projectName,
+	Domain:                  "localhost",
+	PortMin:                 11430,
```
Comment (lmbek) [inline main.go]: Can you change to:
Portmin: 35000
PreferredPort: 35050
PortMax: 35100

**Review by lmbek — CHANGES_REQUESTED** [review]
can you fix the 3 lines, then it is approved

**Comment by notHooman996** [comment]
i fixed it

**Review by lmbek — APPROVED** [review]
approved

**Pull request merged**